avocado/plugins/assets.py

```diff
@@ -208,6 +208,9 @@ def fetch_assets(test_file, klass=None, method=None):
             call['cache_dirs'] = cache_dirs
             call['expire'] = expire
             asset_obj = Asset(**call)
+            if logger is not None:
+                logger.info('Fetching asset from %s:%s.%s',
+                            test_file, klass, method)
```

Comment:
I would suggest to use the new python f-string style:

```
f"Fetching asset from {test_file}, {klass}, {method}"
```

Just as a suggestion!

Reply:
I'm still full of Python 2 (bad) practices, given that we supported both Python 2 and 3 for so long.  I'll try to adopt that in every new code.  Thanks for the reminder/suggestion!

Reply:
Watch out! F-Strings are Python >= 3.6.